app/src/main/res/values/strings.xml

```diff
@@ -1298,6 +1298,8 @@ If there are no signs along the whole street which apply for the highlighted sec
     <string name="quest_bbq_fuel_wood">Wood</string>
     <string name="quest_bbq_fuel_electric">Electric</string>
     <string name="quest_bbq_fuel_charcoal">Charcoal</string>
+    <string name="quest_bbq_fuel_not_a_bbq">"It's a fire enclosure"</string>
+    <string name="quest_bbq_fuel_not_a_bbq_confirmation">"Fire enclosure doesn't have cooking surface (e.g. grate/plate), as opposed to grill."</string>
```

Comment:
1. Please use typographic apostrophes (`’` instead of `'`), then the double quotes around the string are also not needed.
2. I suggest this little message improvement, but I welcome any native speakers to suggest further improvements!

```suggestion
    <string name="quest_bbq_fuel_not_a_bbq">It’s a fire enclosure</string>
    <string name="quest_bbq_fuel_not_a_bbq_confirmation">Fire enclosures don’t have a cooking surface (e.g. grate/plate), as opposed to grills.</string>
```